### PR TITLE
command line via product description - need to incorporate Odds into interface when merged into master

### PR DESCRIPTION
**Pull request by morecallan**
@jcockhren @katerebekah @BlaiseGratton @guthb CLI interaction complete minus incorporation of odds class

### CODE REVIEW

**Comment by guthb** [comment]
@callan, works on my machine, if you don't mind after we have instructor approval can you let me add the new functionality to Odds for a PR that would come from me?
👍

File: Mathmagician/Program.cs [hunk 1]
```diff
@@ -10,7 +10,64 @@ class Program
...
+                string listAmountInput = Console.ReadLine();
+                int listAmountParsed = Int32.Parse(listAmountInput);
+
+                Console.WriteLine("Cool, I'm going to print {0} {1} numbers.", listAmountInput, commandInput);
+
+
+
+                switch (commandInput)
+                {
+                    case "Integers":
```
Comment (jcockhren) [inline Mathmagician/Program.cs]: @morecallan As it is now, this requires the user to type `Integers` exactly. Consider/think of a way to allow for the user to provide case-insensitive responses. Both `Integer` and `integer` should work.

Comment (morecallan) [inline Mathmagician/Program.cs]: @jcockhren total misread of the project instructions on my part. thanks for clarifying.

**Comment by jcockhren** [comment]
:+1:

**Pull request merged**